### DISCUSSION TITLE
[Feature Request]: Downgrading php version

### DISCUSSION THREAD

**Discussion by emredipi** [beyondcode/herd-community#363] [Ideas]
### Feature Description

I was using php 8.2.11 and upgraded my php version to 8.2.13 but having issues with this version. I can't downgrade it to 8.2.11.

### Is this feature valuable for other users as well and why?

When a version has a problem, unless PHP release a new version nobody can fix it. So downgrading is a good solution.

**Comment by rhino-corey** [comment]
Yes. I needed this today since the latest PHP 8.3 broke my site.

https://github.com/php/php-src/issues/16870
